src/sentry/utils/email.py

```diff
@@ -342,6 +348,11 @@ def get_built_messages(self, to=None, bcc=None):
             logger.debug('Did not build any messages, no users to send to.')
         return results
 
+    def format_to(self, to):
+        if to and len(to) > MAX_RECIPIENTS:
```

Comment:
TLDR if we still try to notify on something and no one is subscribed to it, we won't error out.